airflow/providers/alibaba/cloud/example_dags/example_oss_bucket.py

```diff
@@ -14,6 +14,10 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
+# Ignore mypy argument checking. Some operator args will be passed via ``default_args``.
+# type: ignore[call-arg]
```

Comment:
I think instead of ignoring like this, we should probably “fix” `@task` to make these not fail in the first place.

Reply:
We've got airflow.mypy.plugin.decorators doing _some_ things already, so that would be the place to add that sort of check if we cant do it with type stubs etc.

Reply:
Yeah. I fully agree we should use the opportunity to improve our type hinting rather than disable it, as our goal is not to pass `mypy` but to get better type hinting and less errors :)

There is a class of errors that can be avoided if we do it  (for example most of the None`s) - that actually hit our users in the recent  months, so  this might have some long-lasting "real-world" consequences.

Reply:
This one is really about not add "blanket" ignore for whole file, but trying to find better ways to deal with it. Maybe not yet complete, but a way that will be "simple" and easy to apply to get rid of all the current MyPy complaints. 

There is a very interesting parallel discussion on validation of the taskflow args dynamically or at runtime: https://github.com/apache/airflow/issues/19884. but I think we should not go there yet, we should just focus on what MyPy complains on currently.

We also have this one: https://github.com/apache/airflow/pull/19886 by @uranusjr  where we have a stub added to context so that the parameters in context are also type-hinted.

I am quite sure with some clever tricks we will be able to use MyPy to do more type validation on context params and inputs/outputs passed between tasks via task flows but, I think this should be next step after we fix all the current problems.

Reply:
>This one is really about not add "blanket" ignore for whole file, but trying to find better ways to deal with it.

Yeah it felt icky to do it and to slap `# type: ignore[call-args]` on each operator. But I suppose if they felt icky, probably means I shouldn't have done it. Although maybe the latter is a "passable" interim approach until the other conversations/solutions you pointed out get fleshed out? If there is no urgent timeline on this then perhaps it is better to put in a solid fix rather than bandaids.

Reply:
Agree. We have a few parallel changes for that one and hopefully we'll find some good solutions for similar cases.

Reply:
Sorry @potiuk I worded my comment poorly. You agree that I should add the `ignore[call-args]` markers or I should hold off on this PR/defer to the better solution?